deployment/network-operator/README.md

```diff
@@ -204,9 +208,24 @@ devicePlugin:
   deploy: true
   reources:
     - name: rdma_shared_device_a
-      devices: [enp1, enp2]
+      ifNames: [enp1, enp2]
     - name: rdma_shared_device_b
-      devices: [ib0]
+      ifNames: [ib0]
+```
+
+#### Example 3
```

Comment:
you now have two Example 3 in the doc, also example 3 below is outdated

Reply:
fixed